client/.eslintrc

```diff
@@ -6,13 +6,46 @@ extends: eslint-config-airbnb
 plugins:
   - react
 
+globals:
+  __DEBUG_SERVER_ERRORS__: true
+  __SERVER_ERRORS__: true
+  google: true
+  amenitiesList: true
+  File: true
```

Comment:
Let's remove `google`, `amenitiesList` & `File` from here, it's project specific.